src/Resources/skeleton/resetPassword/twig_request.tpl.php

```diff
@@ -0,0 +1,22 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Reset your password{% endblock %}
+
+{% block body %}
+    {% for flashError in app.flashes('reset_password_error') %}
+        <div class="alert alert-danger" role="alert">{{ flashError }}</div>
+    {% endfor %}
+    <h1>Reset your password</h1>
+
+    {{ form_start(requestForm) }}
+    {{ form_row(requestForm.<?= $email_field ?>) }}
```

Comment:
Can we indent this 4 more spaces so it's inside the `{{ form_start() }}`?

Reply:
done